lib/upstream/zookeeper.js

```diff
@@ -14,10 +14,9 @@ class Zookeeper extends EventEmitter {
     super();
     this._conf = conf;
 
-    var zkPort = this._conf.zk.port;
-    var zkHost = this._conf.zk.host;
+    var {host, port} = this._conf.zk;
 
-    this._client = zookeeper.createClient(`${zkHost}:${zkPort}`);
+    this._client = zookeeper.createClient(`${host}:${port}`);
```

Comment:
Because we're in a `Zookeeper` class and accessing a `zk` object, I think we can safely drop the prefix from these variable names and use simple destructuring assignment.